docs/integrations/optuna.rst

```diff
@@ -43,22 +45,33 @@ Monitor the metrics and parameters checked at each run:
 .. code-block:: python3
 
     study = optuna.create_study(direction='maximize')
-    study.optimize(objective, n_trials=100, callbacks=[monitor])
+    study.optimize(objective, n_trials=100, callbacks=[neptune_callback])
 
 Log all results
 ---------------
+You can log additional information from optuna study after the sweep has completed.
+By running:
+
+.. code-block:: python3
 
-The script logs the following to Neptune:
+    opt_utils.log_study_info(study)
+
+You log the following things to Neptune:
 
 * Best score
 * Best parameters
-* plot_convergence figure
-* plot_evaluations figure
-* plot_objective figure
+* Interactive plotly figures from optuna.visualization: plot_contour, plot_slice, plot_parallel_coordinate, optimization_history
+* Pickled study object
+
+Log study and charts after every iteration
+------------------------------------------
+You can log interactive charts from optuna.visualization and the study object after every iteration.
+Simply specify what you want to log at `NeptuneCallback` creation.
 
 .. code-block:: python3
 
-    opt_utils.log_study(study)
+    neptune_callback = opt_utils.NeptuneCallback(log_study=True, log_charts=True)
```

Comment:
Maybe the defaults should be like this. You log everything by default. Then this call is simpler.

Reply:
Especially for shorter runs you may not want to (as it slows down the process).
Also, I figured I don't want people using it to feel the difference with update of contribs.
Idk, I'd rather leave it as is.